### PR TITLE
added loading of an existing coop incorporation

### PR DESCRIPTION
**Pull request by thorwolpert**
*Issue #:* /bcgov/entity#7158

*Description of changes:*
- loading an existing incorporation, will create the entity in LEAR

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Comment by thorwolpert** [comment]
CodeSmell: Not refactoring for complexity yet.

File: queue_services/entity-filer/src/entity_filer/filing_processors/incorporation_filing.py [hunk 1]
```diff
@@ -126,13 +131,13 @@ def process(business: Business, filing: Dict, filing_rec: Filing):
...
         shares.update_share_structure(business, share_structure)
 
     if name_translations := incorp_filing.get('nameTranslations'):
         aliases.update_aliases(business, name_translations)
 
-    if not is_correction:
+    if not is_correction and not filing_rec.colin_event_ids:
         # Update the filing json with identifier and founding date.
         ia_json = copy.deepcopy(filing_rec.filing_json)
         ia_json['filing']['business']['identifier'] = business.identifier
```
Comment (jordiwes) [inline queue_services/entity-filer/src/entity_filer/filing_processors/incorporation_filing.py:143]: just curious if we need to copy anything else from colin (like founding date), or is this done elsewhere?

Comment (thorwolpert) [reply queue_services/entity-filer/src/entity_filer/filing_processors/incorporation_filing.py:143]: yes, that will be the filings effective date, for setting the founding date of the existing corp

Comment (jordiwes) [reply queue_services/entity-filer/src/entity_filer/filing_processors/incorporation_filing.py:143]: gotcha

**Review by jordiwes — APPROVED** [review]
Looks good

**Pull request merged**